### PR TITLE
Core/Spell: Remove Rigor Mortis casting on DK creation and make Rigor…

### LINKED ISSUE #15248: [6.x] Undead starting problems regarding Rigor Mortis
I noted a bug in undead character start, regarding the "debuff" that you get once you create character and login called [Rigor Mortis](http://www.wowhead.com/spell=73523/rigor-mortis) spell.

Details:
* Non DK characters:
    * RETAIL: Rigor mortis is a buff (appears in the top side with the "good" buffs), and is dispelleable by user by making right click on it
    * TC: Rigor mortis is a debuff (appears a little down of the other buffs), and is **not** dispelleable by user by making right click on it
* DK characters:
    * RETAIL: Rigor mortis is not present at all
    * TC: Rigor mortis is a debuff, like above

In this case, the bug is important, as this makes DK character unplayable (because in non DK, the first quest you can take, removes you Rigor Mortis correctly, but that quest is not present in DK zone, so you can't simply move)

Tested in rev 590c6e399d778db459fc8417322c80549081440a

### PR DESCRIPTION
**Pull request by StormBytePP**
… Mortis spell be positive instead of negative.
Fix by @Killyana and @killradio
Closes #15248

### CODE REVIEW

File: src/server/game/Spells/SpellInfo.cpp [hunk 1]
```diff
@@ -2918,6 +2918,7 @@ bool SpellInfo::_IsPositiveEffect(uint32 effIndex, bool deep) const
                 case 62344: // Fists of Stone
                 case 61819: // Manabonked! (item)
                 case 61834: // Manabonked! (minigob)
+		case 73523: // Rigor Mortis
```
Comment (Kittnz) [inline src/server/game/Spells/SpellInfo.cpp]: tabs

**Comment by StormBytePP** [comment]
@Kittnz fixed in last commit (also squashed the two commits into one)

**Comment by Walkum** [comment]
Unblizzlike. On retail the spell must be removed when accept the first quest of the val'kyr, not removable by yourself.

**Comment by pete318** [comment]
I just rolled an undead warlock. You can remove the spell yourself with no problem whatsoever.

**Comment by killradio** [comment]
LOL thx for mention @Killyana and me in your pull request, thats my boy!

**Comment by StormBytePP** [comment]
@Walkum You can remove by yourself in retail *BEFORE* taking the quest. I tried.
Also, it is removed automatically with the quest.

**Comment by StormBytePP** [comment]
@killradio PR still don't have green light to be accepted, and also it may require further commits to be squashed later. That is why I did not mentioned both of you.
Specially because I am not sure of the naming of the SQL update file, but be sure that once I get the green light, I will of course do!

**Comment by jackpoz** [comment]
is this a 6.x only PR ?

**Comment by Killyana** [comment]
Yes 6.x only

**Pull request merged**